packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+      break;
+    case 6:
+      res.push('col-span-6') 
+      break;
+    case 7:
+      res.push('col-span-7') 
+      break;
+    case 8:
+      res.push('col-span-8') 
+      break;
+    case 9:
+      res.push('col-span-9') 
+      break;
+    case 10:
+      res.push('col-span-10') 
+      break;
+    case 11:
+      res.push('col-span-11') 
+      break;
+    case 12:
+      res.push('col-span-12') 
+      break;
+    default:
+      break;
+  }
+  return res;
+})
+
+const headerStaticClasses = ['fz__header__cell','sticky','top-0','z-[2]', 'bg-grey-100', 'px-16', 'h-48', 'font-medium', 'flex', 'justify-start', 'items-center']
+const headerClasses = computed(() => {
+
+})
+
+const bodyStaticClasses = ['fz__body', 'z-[1]', 'px-16', 'h-48', 'bg-core-white', 'flex', 'justify-start', 'items-center', 'min-w-min', 'border-b-1', 'border-grey-100']
+const bodyClasses = computed(() => {
+
+})
+
+const getBodyClasses = (column: {props: FzColumnProps, children: FzColumnSlots}, isHeader?: boolean) => {
+  return {
+    relative: !column.props.sticky,
+    sticky: column.props.sticky,
+    ['left-0 z-[2]']: column.props.sticky === 'left',
+    ['z-[3]']: column.props.sticky && isHeader,
+    ['right-0']: column.props.sticky === 'right'
+  }
+}
+
+const updateClasses = () => {
+  const tableWidth = columns.reduce((acc, el, index) => {
+    const colWidth = grid.value.children[index].getBoundingClientRect().width
+    acc+=colWidth
+    return acc
+  }, 0)
+  grid.value.style.width = `${tableWidth}px`
+}
+
+const centerPageList = computed(() => {
+  if (!props.pages) {
+    return []
+  }
+  const safeActivePage = activePage.value || 0
+  return new Array(props.pages)
+    .fill(0)
+    .map((val, index) => index)
+    .filter(el => {
+      const res = (Math.abs(el - safeActivePage) <= props.pageInterval) && el !== 0 && el !== props.pages - 1
+      return res
+    })
+})
+
+onMounted(() => {
+  updateClasses()
+
+  if (props.actions?.items.length) {
+
+  }
+})
+onUpdated(updateClasses)
+</script>
+
+<template>
+<div class="fz__table overflow-auto h-full w-full">
+  <div
+    :class="[staticClasses, classes]"
+    ref="grid"
+    role="table"
+    :aria-rowcount="value.length"
+    :aria-colcount="columns.length">
+    <template  v-for="(column, index) in columns">
+      <div :class="[headerStaticClasses, headerClasses, getBodyClasses(column, true)]" role="columnheader" aria-sort="none" >
+         {{ column.props.header }} 
+      </div>
+    </template>  
+    <div
+      v-if="actions"
+      :class="['fz__table__header--actions', headerStaticClasses, headerClasses]">
+      Azioni
+    </div>
+    <template v-if="value && value.length" v-for="(row, index) in value">
+      <div
+        :aria-rowindex="index"
+        :class="[bodyStaticClasses, bodyClasses, getBodyClasses(column)]"
+        role="cell"
+        v-for="column in columns">
+        {{ row[column.props.field?.toLowerCase() || column.props.header.toLowerCase()] }}
```

Comment:
```suggestion
        {{ (row[column.props.field] || column.props.header)?.toLowerCase() }
```
but why is the header used as fallback?